CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -850,9 +850,6 @@ CF_INLINE const char *CFPathRelativeToAppleFrameworksRoot(const char *path, Bool
 #if (DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_EMBEDDED_MINI || DEPLOYMENT_TARGET_LINUX)
 #if __HAS_DISPATCH__
 
-#include <dispatch/dispatch.h>
-#include <dispatch/private.h>
```

Comment:
why don't we have /usr/local/include/dispatch/private.h? can we just pass across private header paths if it isn't installed into a staging sysroot?

and I am not certain that /usr/include/dispatch/dispatch.h missing is a good thing. If that is not actually missing then perhaps we should just move the private.h out only.